scripts/bin/cps_api_db_service.sh

```diff
@@ -13,8 +13,8 @@
 # See the Apache Version 2.0 License for specific language governing
 # permissions and limitations under the License.
 
-#export PYTHONPATH=/opt/dell/os10/lib:/opt/dell/os10/lib/python
-#/usr/bin/python /opt/dell/os10/lib/cps_db_stunnel_manager.py
+#export PYTHONPATH=/usr/lib/opx:/usr/lib/x86_64-linux-gnu/opx
+#/usr/bin/python cps_db_stunnel_manager.py
 
```

Comment:
@ArthiSivanantham, this is commented out, so it seems little reason to update paths -- better to remove it entirely.  Also, the following comment suggests that the file may be able to be removed completely.